stwcs/updatewcs/astrometry_utils.py

```diff
@@ -150,11 +157,11 @@ def __init__(self, url=None, raise_errors=None, perform_step=True,
         # otherwise, it will turn off raising Exceptions
         #
         self.raise_errors = False
+        if pipeline_error_envvar in os.environ:
+            self.raise_errors = True
```

Comment:
It was never clear to me what the allowed values of this env var are. The docs say it's a boolean so I assume it can be set to True or False. But then the code tests only that it is set.